File: app/src/main/java/com/android/test/seaworld/model/animals/Tux.java
package com.android.test.seaworld.model.animals;


import com.android.test.seaworld.R;
import com.android.test.seaworld.model.SeaWorldModel;
import com.android.test.seaworld.utils.Settings;

public class Tux extends Animal{



    public Tux(SeaWorldModel seaWorldModel){
        super(seaWorldModel);
        timeToReprodution = Settings.getTimeToReproductionTux();
    }

    @Override
    public int getDrawableResourceId() {
        return R.drawable.tux;
    }
}

File: app/src/main/java/com/android/test/seaworld/model/animals/Tux.kt
package com.android.test.seaworld.model.animals


import com.android.test.seaworld.R
import com.android.test.seaworld.model.SeaWorldModel
import com.android.test.seaworld.utils.Settings

class Tux(seaWorldModel: SeaWorldModel) : Animal(seaWorldModel) {

    override val animalImageResource: Int
        get() = R.drawable.tux

    init {
        timeToReprodution = Settings.timeToReproductionTux
    }
}
